systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -363,6 +363,11 @@ public interface Constants {
      */
     String ROUTE = "route";
 
+    /**
+     * Tag for tests, without ARM,AARCH64 support
+     */
+    String ARM_UNSUPPORTED = "armunsupported";
```

Comment:
Would `ARM64_UNSUPPORTED` and `arm64unsupported` make more sense? In case we have some other Arm platforms in the future :-o?